### PR TITLE
Update TuneMe to Molo 3 and django 1.9 [SEED-461]

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review :)

File: docker/docker-start.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@ set -e
 # ensure we create the locales first so that Django doesn't complain
 LANGUAGE_CODE=en ./manage.py compilemessages
 
+./manage.py migrate core 0031 --fake
```
Comment (miltontony) [inline docker/docker-start.sh]: This is not going to work - for old sites, yes, we need to do this, but for new sites, we can't.
Please remove this and only add it to the `sideloader/postinstall`

File: tuneme/settings/base.py [hunk 2]
```diff
@@ -133,11 +164,30 @@
...
+        'schedule': crontab(minute=0),
+    },
+}
+CELERYBEAT_SCHEDULER = "djcelery.schedulers.DatabaseScheduler"
 
 # Internationalization
 # https://docs.djangoproject.com/en/1.7/topics/i18n/
 
-LANGUAGE_CODE = 'en-gb'
+LANGUAGE_CODE = environ.get('LANGUAGE_CODE', 'en-gb')
```
Comment (miltontony) [inline tuneme/settings/base.py]: This should be `en`

**Comment by miltontony** [comment]
👍

**Comment by miltontony** [comment]
👍

**Pull request merged**